documentation/modules/auxiliary/gather/qnap_lfi.md

```diff
@@ -0,0 +1,119 @@
+## Introduction
+
+This module abuses a vulnerability in QNAP QTS and PhotoStation that allows an
+unauthenticated user to download files off the file system, and because the server
+runs as root, it's possible to include sensitive files, including ssh private keys and
+password hashes.
+
```

Comment:
i know there is some CVE confusion since 2 have the same description, and the 3rd may be it, but may not? You may want to write a note in here about that